valerymo/integreatly-operator
# MGDAPI-1882 - Update/Create automated e2e tests/test cases - DO NOT Merge. Deprecated

This PR is deprecated. New PR based on master created: https://github.com/integr8ly/integreatly-operator/pull/2188
# Issue link
https://issues.redhat.com/browse/MGDAPI-1882

# What
Automation tests update for Observability Operator.
AMO Automation tests - continue work (will be removed in different Task/PR). Some AMO Automation tests  code was impacted, to allow similar tests both for AMO and Observability Operator meanwhile.

# Verification steps
- Check that new Automation Tests (named as E00**) are running (command line single tests)
- Check that AMO  Automation Tests continue run successfully  (command line single tests)
- New automated test case running in pipeline without errors
- AMO automated test case running in pipeline without errors
